drivers/netdev_ieee802154_submac/netdev_ieee802154_submac.c

```diff
@@ -61,6 +61,12 @@ static int _get(netdev_t *netdev, netopt_t opt, void *value, size_t max_len)
         case NETOPT_STATE:
             *((netopt_state_t*) value) = _get_submac_state(submac);
             return 0;
+        case NETOPT_TX_RETRIES_NEEDED:
```

Comment:
this could be extended to be used with hardware retransmission too (maybe copying it to an internal variable).

Reply:
Ah good catch.
I was under the impression that the `NETOPT_TX_RETRIES_NEEDED` would just be handed on to the hardware netdev driver in the `ieee802154_radio_has_frame_retrans() case,  which would then handle it - but there is no other netdev driver.

Reply:
This is the one: https://github.com/RIOT-OS/RIOT/blob/master/sys/include/net/ieee802154/radio.h#L1112

If that feature is available, this should be copied somewhere during TX_COMPLETE. Otherwise, if there's no hardware retransmission, we fallback to these lines here.
In case there's HW retransmissions but no info, we cannot show it anyway (maybe `-1` could be used to indicate such case)